src/modules/design/bpmn-diff-view/bpmn-diff-view.ts

```diff
@@ -73,6 +74,7 @@ export class BpmnDiffView {
   private _subscriptions: Array<Subscription>;
   private _elementNameService: ElementNameService;
   private _diffDestination: string = 'lastSaved';
+  private _diagramName: string;
```

Comment:
I would recommend the following, because that variable is not always a string and knowingly set to `undefined` at times.
```suggestion
  private _diagramName: string | undefined;
```
TypeScript has a strict type which would enforce this. Even if it's too late to introduce that mode, I think that type would give you a better understanding of that variables behaviour.